common/page.go

```diff
@@ -874,7 +878,13 @@ func (p *Page) Goto(url string, opts *FrameGotoOptions) (*Response, error) {
 	)
 	defer span.End()
 
-	return p.MainFrame().Goto(url, opts)
+	resp, err := p.MainFrame().Goto(url, opts)
+	if err != nil {
+		SpanRecordError(span, "goto failed", err)
```

Comment:
Would it make sense to add some context to the error here?

Reply:
I believe the inner code already adds the necessary context: https://github.com/grafana/xk6-browser/blob/e4c6ed3a612ffe5f395804e825b2010800d7bc24/common/frame.go#L973-L987.

Reply:
Yeah, exactly, the called method already returns the full context of the issue. I can wrap it in something like `err := fmt.Errorf("navigating page to %q: %w", url, err)`, to help identify that it was `page` calling `frame`, but it feels repetitive. WDYT?

Reply:
I think the existing error is good enough.